CONTRIBUTING.md

```diff
@@ -43,28 +43,28 @@ you can follow the steps below to test your changes:
 
 1. Create the cluster:
 
-```sh
-kind create cluster operator-controller
-```
+    ```sh
+    kind create cluster operator-controller
```

Comment:
It was rendering fine in GH UI. However this doc is also used in the website. This formatting works fine in both GH and on the website.